gdnative-sys/build.rs

```diff
@@ -12,20 +12,21 @@ fn main() {
 mod header_binding {
     use std::path::PathBuf;
 
-    #[cfg(any(target_os = "macos", target_os = "ios"))]
     fn osx_include_path() -> Result<String, std::io::Error> {
         use std::process::Command;
 
         let output = Command::new("xcode-select").arg("-p").output()?.stdout;
         let prefix_str = std::str::from_utf8(&output).expect("invalid output from `xcode-select`");
-        let prefix = prefix_str.trim_right();
+        let prefix = prefix_str.trim_end();
 
-        let platform = if cfg!(target_os = "macos") {
+        let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
+
+        let platform = if target_os == "macos" {
             "MacOSX"
-        } else if cfg!(target_os = "ios") {
+        } else if target_os == "ios" {
```

Comment:
I'm late to the party but have been messing around with rust on iOS for a bit lately and in order to better handle #343, I think you'll want to do something like [my WIP uikit-sys crate](https://github.com/simlay/uikit-sys/blob/36686b858d013c6856ec16271113251c02ed746b/build.rs#L3-L25). I think it's a bit more idiomatic to use `xcrun` to get the include path.

Reply:
Thanks for the suggestion! I have created a separate issue for this improvement: #352. A PR would be great if you'd like to create one!